src/webview/memory-webview-view.tsx

```diff
@@ -81,21 +87,22 @@ class App extends React.Component<{}, MemoryAppState> {
                 offset={this.state.offset ?? 0}
                 count={this.state.count}
                 updateMemoryArguments={this.updateMemoryState}
+                updateMemoryDisplayConfiguration={this.updateMemoryDisplayConfiguration}
+                resetMemoryDisplayConfiguration={this.resetMemoryDisplayConfiguration}
                 refreshMemory={this.refreshMemory}
-                toggleColumn={this.toggleColumn}
                 fetchMemory={this.fetchMemory}
                 isMemoryFetching={this.state.isMemoryFetching}
+                groupsPerRow={this.state.groupsPerRow}
+                wordsPerGroup={this.state.wordsPerGroup}
+                scrollingBehavior={this.state.scrollingBehavior}
+                visibleColumns={this.state.visibleColumns}
```

Comment:
We are also passing down `this.state`columns`, which should be marked `active` or not - is the double bookkeeping necessary, or how could we consolidate it better?